### DISCUSSION TITLE
Survey on Package Management Software in 2024

### DISCUSSION THREAD

**Discussion by shogo4405** [HaishinKit/HaishinKit.swift#1488] [Polls]
Currently, HaishinKit supports SPM, CocoaPods, and Carthage. To streamline maintenance, I'm considering unifying to SPM. I would like to take your usage into account when deciding on package management software support. Please vote for the package management software you use to manage HaishinKit.

**Comment by levs42** [comment]
Maybe it's different now, but last time I was checking it was pain to add changes to a dependency using SPM. In CocoaPods I just specify a local folder. With SPM people do "drag and drop to XCode" which seems to be broken in XCode 15 anyway.

**Comment by shogo4405** [comment]
Thank you to everyone who voted. I plan to discontinue Carthage support in the next major release. Thank you for your support.
Yesterday, the [CocoaPods Support & Maintenance Plans](https://blog.cocoapods.org/CocoaPods-Support-Plans/) were announced.
